src/ApparentHorizons/StrahlkorperGr.cpp

```diff
@@ -202,20 +202,22 @@ Scalar<DataVector> spin_function(
       make_with_value<Scalar<DataVector>>(area_element, 0.0);
   for (size_t i = 0; i < 3; ++i) {
     for (size_t j = 0; j < 3; ++j) {
+      const DataVector extrinsic_curvature_normal_sin_theta =
```

Comment:
here extrinsic.... += K(i,j) * n(j) . (we are going to move the sin_theta multiplication to the end)